app/assets/javascripts/app/discussions/discussion.js

```diff
@@ -40,10 +40,14 @@
 
     function toggleCommentForm() {
       if(Device.isMobile()) {
+        if (ctrl.showNewComment === true) {
+          $rootScope.$broadcast('comment_focusOut');
```

Comment:
why are you mixin snake_case with camelCase? Use something like `comment:close`.